build/azure-pipelines/win32/sql-product-build-win32.yml

```diff
@@ -27,7 +27,7 @@ steps:
   - powershell: |
       . build/azure-pipelines/win32/exec.ps1
       $ErrorActionPreference = "Stop"
-      exec { tar --force-local -xzf $(Pipeline.Workspace)/compilation.tar.gz }
+      exec { tar -xf $(Pipeline.Workspace)/compilation.tar.gz }
```

Comment:
on Windows 2019, there is a tar.exe shipped with Windows that do not support those parameter, I am adjusting it to work with the new tar.exe